src/SDKs/DataFactory/DataFactory.Tests/JsonSamples/DatasetJsonSamples.cs

```diff
@@ -402,5 +402,71 @@ public class DatasetJsonSamples : JsonSampleCollection<DatasetJsonSamples>
     }
 }
 ";
+        [JsonSample]
```

Comment:
Thanks for the samples, but I don't see any actual test updates

Reply:
these run automatically as part of `LinkedService_SerializationTest`, `Dataset_SerializationTest` and `Activity_SerializationTest`

Reply:
see the `ClassData` attribute on each